src/app/core/app.component.scss

```diff
@@ -73,7 +73,16 @@
     padding-top: 7.5rem;
   }
 
-  &.collapsed, &.mobile {
```

Comment:
So don't wee need to hide title on mobile? https://github.com/France-ioi/AlgoreaFrontend/pull/1366#issuecomment-1540380333

Reply:
Hum I need to test a bit.

Reply:
From current "Sketch"es , no.